website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Connecting your GitLab Account"
+id: "connecting-gitlab"
+sidebar_label: "Connecting your GitLab Account"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, let us know at sales@getdbt.com.
+
+:::
+
+## Overview
+
+Connecting your GitLab account to dbt Cloud unlocks exciting and compelling functionality in dbt Cloud. Once your GitLab account is connected, you can:
+- trigger CI builds when Pull Requests are opened in GitLab!
+- log into dbt Cloud via oauth through GitLab!
+- add new repos to Cloud in a single click!
+- and more!
+
+## Linking dbt Cloud to your GitLab account
+
+To link your dbt Cloud account to your GitLab account, navigate to your Account Settings and click the Integrations tab. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/gitlab_navigation_setup.gif" title="Like so!"/>
```

Comment:
I'm not _great_ about this, but we should use titles that explain what is represented in the image. This makes the webpage more accessible to users that use screenreaders, for instance. Can we change this title to better describe the depiction in the gif?